nservicebus/transactional-session/index.md

```diff
@@ -51,17 +50,45 @@ Sending messages in an atomic manner is done through the `ITransactionalSession`
 
 snippet: sending-transactional-session
 
-The persistence specific database session is accessible via the `transactionalSession.SynchronizedStorageSession` property or via dependency injection. See the [persistence specific documentation](/nservicebus/transactional-session/persistences) for more details.
+The persistence-specific database session is accessible via the `transactionalSession.SynchronizedStorageSession` property or via dependency injection. See the [persistence-specific documentation](/nservicebus/transactional-session/persistences) for more details.
 
 Once all the operations that are part of the atomic request have been executed, the session should be committed:
 
 snippet: committing-transactional-session
 
-Disposing the transactional session without committing, will roll-back any changes that were made.
+Disposing the transactional session, without committing, will roll back any changes that were made.
+
+### Advanced
```

Comment:
```suggestion
### Advanced settings
```

Reply:
this could also be a `##` section?

Reply:
I placed it as a subsection to the usage settings and that's why it only says "advanced" in reference to "advanced usage"